src/cmd-init

```diff
@@ -148,5 +162,9 @@ if test "${TRANSIENT}" = 1; then
     touch tmp/cosa-transient
 fi
 
+if [ -n "${EXTREPOS}" ]; then
```

Comment:
Not necessary to start, but it'd be nice to support local paths for this too like we do with the main src config repo.